data/856/677/93/85667793.geojson

```diff
@@ -62,6 +62,9 @@
         "\u039d\u03bf\u03bc\u03cc\u03c2 \u03a3\u03ba\u03cc\u03b4\u03c1\u03b1\u03c2"
     ],
     "name:eng_x_preferred":[
+        "Shkod\u00ebr"
```

Comment:
It's fine to have accent marks here in `name:eng_x_preferred`, but down in `wof:name` did we receive an ASCII value to use instead?

Reply:
This is resolved in 3d28814